scripts/install

```diff
@@ -204,6 +204,11 @@ def main():
             print("You can now run: %s --version" % opts.bin_location)
         else:
             print("You can now run: %s --version" % real_location)
+        print('Note: AWS CLI version 2, the latest major version '
```

Comment:
It might be nice to begin this with a newline. Otherwise, the usage and the note are too close together and a bit hard to read especially for smaller terminals.